demo/src/app/components/datepicker/demos/range-popup/datepicker-range-popup.html

```diff
@@ -43,7 +45,9 @@
              [value]="formatter.format(toDate)"
              (input)="toDate = validateInput(toDate, dpToDate.value)">
       <div class="input-group-append">
-        <button class="btn btn-outline-secondary calendar" (click)="datepicker.toggle()" type="button"></button>
+        <button class="btn btn-outline-secondary calendar" (click)="datepicker.toggle()" type="button">
+          <span class="sr-only">Choose date</span>
```

Comment:
Might be worth thinking about doing these as `aria-label`s instead so there's fewer instances of `sr-only` to rename when going to BS5.